schema/spacewalk/susemanager-schema.changes

```diff
@@ -1,4 +1,17 @@
+- Prevent SELECT INSTR error in Postgres logs every minute (bsc#1157034)
 - Migration paths from 3.2.20 and 3.2.22 to 4.0.0
+
+-------------------------------------------------------------------
+Thu Oct 31 15:12:19 CET 2019 - jgonzalez@suse.com
+
+- version 4.0.16-1
+- Fix: regression with Ubuntu version compare (bsc#1150113)
+
+-------------------------------------------------------------------
+Thu Oct 17 13:12:44 CEST 2019 - jgonzalez@suse.com
+
+- version 4.0.15-1
+>>>>>>> efc8f462f6f... Prevent SELECT INSTR error in Postgres logs every minute (bsc#1157034)
```

Comment:
Conflict! Please add only your line on top of the file.

Reply:
Argh! fixed